webserver/templates/user/scraper.js

```diff
@@ -148,7 +148,10 @@ function getLastFMPage(page) {
 
 var version = "1.4";
 var page = 1;
-var numberOfPages = parseInt(document.getElementsByClassName("pages")[0].innerHTML.trim().split(" ")[3]);
+var numberOfPages = 1;
+if (document.querySelector('.pages') != null) {
```

Comment:
I'm not sure about the difference between these two usages - what does `document.getElementsByClassName` and `document.querySelector` do differently? My preference would be to use the same one in both places. e.g. you could do 
```javascript
var pages = document.getElementsByClassName("pages");
if (pages !== null) {
  numberPages = pages.something.something;
}
```